week1/645_set_mismatch.py

```diff
@@ -0,0 +1,11 @@
+# https://leetcode.com/problems/set-mismatch/
+def set_mismatch(nums: list[int]) -> list[int]:
+    freq_dict = {i + 1: 0 for i in range(len(nums))}
```

Comment:
Using a set might serve better purpose if you want to check the existence of a value in a collection, and while iterating the loop in line 4 you can check whether that value already exists.